From a6b4da7ba862999dfddeaa9ef164bfbe79b47d81 Mon Sep 17 00:00:00 2001
From: allen yan <yanwei@marvell.com>
Date: Wed, 12 Apr 2017 15:26:29 +0800
Subject: [PATCH 0176/1239] arm64: config: a3700: enable board information
 display

This patch enables CONFIG_DISPLAY_BOARDINFO config to print
board information during boot.

Change-Id: I58d63b12f51f76f2293b5e6e2cfe7e117060b903
Signed-off-by: allen yan <yanwei@marvell.com>
---
 configs/mvebu_db-88f3720_defconfig          | 1 -
 configs/mvebu_espressobin-88f3720_defconfig | 1 -
 2 files changed, 2 deletions(-)

diff --git a/configs/mvebu_db-88f3720_defconfig b/configs/mvebu_db-88f3720_defconfig
index bd99fbb369..b1bae26781 100644
--- a/configs/mvebu_db-88f3720_defconfig
+++ b/configs/mvebu_db-88f3720_defconfig
@@ -16,7 +16,6 @@ CONFIG_USE_PREBOOT=y
 CONFIG_SYS_CONSOLE_ENV_OVERWRITE=y
 CONFIG_SYS_CONSOLE_INFO_QUIET=y
 # CONFIG_DISPLAY_CPUINFO is not set
-# CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_ARCH_EARLY_INIT_R=y
 CONFIG_BOARD_EARLY_INIT_F=y
diff --git a/configs/mvebu_espressobin-88f3720_defconfig b/configs/mvebu_espressobin-88f3720_defconfig
index 1a1c0b7e0e..ae0323e349 100644
--- a/configs/mvebu_espressobin-88f3720_defconfig
+++ b/configs/mvebu_espressobin-88f3720_defconfig
@@ -16,7 +16,6 @@ CONFIG_USE_PREBOOT=y
 CONFIG_SYS_CONSOLE_ENV_OVERWRITE=y
 CONFIG_SYS_CONSOLE_INFO_QUIET=y
 # CONFIG_DISPLAY_CPUINFO is not set
-# CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_ARCH_EARLY_INIT_R=y
 CONFIG_BOARD_EARLY_INIT_F=y
-- 
2.29.0

